src/mudblazor/MudBlazor.Template/Components/App.razor

```diff
@@ -5,61 +5,43 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <base href="/" />
-    @*#if (IndividualLocalAuth)
+    @*#if (SampleContent)
     <link rel="stylesheet" href="bootstrap/bootstrap.min.css" />
     ##endif*@
-	
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet" />
     <link href="_content/MudBlazor/MudBlazor.min.css" rel="stylesheet" />
-	
-	@*#if (IndividualLocalAuth)
-	<link rel="stylesheet" href="app.css" />
-	##endif*@
-	
+    <link rel="stylesheet" href="app.css" />
```

Comment:
If all remaining Auth pages are upgraded to MudBlazor components, then `app.css` could probably be an empty file.